docs/griptape-framework/structures/workflows.md

```diff
@@ -34,25 +36,19 @@ world_task = PromptTask(
     },
     id="world"
 )
-
-character_task_1 = character_task("scotty", "Scotty")
-character_task_2 = character_task("annie", "Annie")
+workflow.add_task(world_task)
 
 story_task = PromptTask(
     "Based on the following description of the world and characters, write a short story:\n{{ parent_outputs['world'] }}\n{{ parent_outputs['scotty'] }}\n{{ parent_outputs['annie'] }}",
     id="story"
 )
+workflow.add_task(story_task)
 
-workflow = Workflow()
-
-workflow.add_task(world_task)
-
-world_task.add_child(character_task_1)
-world_task.add_child(character_task_2)
-world_task.add_child(story_task)
+character_task_1 = character_task("scotty", "Scotty")
+workflow.insert_task(parent_task, story_task, character_task_1)
```

Comment:
I don't think we've defined `parent_task` at this point in the example

Reply:
Even better, `parent_task` doesn't ever exist. Fixed to `world_task`.